dxr/config.py

```diff
@@ -204,6 +205,7 @@ def __init__(self, name, unvalidated_tree, sections, config):
             Optional('description', default=''): basestring,
             Optional('disabled_plugins', default=plugin_list('')): Plugins,
             Optional('enabled_plugins', default=plugin_list('*')): Plugins,
+            Optional('es_index', default=config.es_index): basestring,
```

Comment:
Huh, I guess we can get away with referencing `config` directly in the `Optional`, if we assume the Config to be immutable (which it is, for now). So this is fine. (Note that we couldn't reference `self`, though, since a missing attribute would crash us during Schema instantiation, before we ever got the opportunity to raise a nice ConfigError.)